src/simweights/_genie_weighter.py

```diff
@@ -55,6 +55,7 @@ def GenieWeighter(file_obj: Any) -> Weighter:  # noqa: N802
     weighter = Weighter([file_obj], surface)
     weighter.add_weight_column("pdgid", weighter.get_column("I3GenieResult", "neu").astype(np.int32))
     weighter.add_weight_column("energy", weighter.get_column("I3GenieResult", "Ev"))
+    weighter.add_weight_column("cos_zen", weighter.get_column("I3GenieResult", "pzv"))
```

Comment:
you need to add a column to the fake data if you want the tests to pass

Reply:
https://github.com/icecube/simweights/blob/b4a664f51698223e9af6eb360270f3e3678c38fb/tests/test_genie_weighter.py#L25